recipe/meta.yaml

```diff
@@ -9,20 +9,19 @@ source:
   sha256: d77dcfa183fb74fa93dce91f3b762cc9dc316c58598112c508a2ea2d873cfcf0
 
 build:
-  number: 0
+  noarch: python
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
 
   run:
-    - python
-    - enum34  # [py<34]
+    - python >=3.6
```

Comment:
This is not considered as a break as if one were to check the previously published releases on conda-forge, you'll notice they were available for py36/7/8/9/10 -> so we're not officially dropping Python 2.7 now by making this happen.